sdk/KoreBuild/KoreBuild.Common.props

```diff
@@ -51,13 +51,7 @@ Default layout and configuration.
 
   <!-- Create a temporary build number if not assigned by the CI server -->
   <PropertyGroup Condition=" '$(BuildNumber)' == '' ">
-    <!--
-      Create second-based build number for local builds.
-      635556672000000000 is Jan 1, 2015.
-    -->
-    <_SecondBasedTimeStamp>$([System.DateTime]::UtcNow.Subtract($([System.DateTime]::FromBinary(635556672000000000))).TotalSeconds.ToString("F0"))</_SecondBasedTimeStamp>
-    <_SecondBasedTimeStamp>t$([System.Int64]::Parse($(_SecondBasedTimeStamp)).ToString("x9"))</_SecondBasedTimeStamp>
-    <BuildNumber>$(_SecondBasedTimeStamp)</BuildNumber>
```

Comment:
Incremental build means something else in MSBuild. What about `IncrementalVersion`?